src/pages/login-helper.js

```diff
@@ -1,49 +1,78 @@
-import { jwtDecode } from 'jwt-decode';
+// login-helper.js
+const jwtDecode = require('jwt-decode');
+
+let decoded;
 
 function authenticate(jwt, cb) {
-    if (typeof window !== "undefined") {
-        sessionStorage.setItem('jwt', jwt);
+  if (typeof window !== "undefined") {
+    const existingJwt = sessionStorage.getItem('jwt');
+    const existingUsers = JSON.parse(sessionStorage.getItem('users')) || [];
 
-        let decoded = jwtDecode(jwt);
-        sessionStorage.setItem('username', decoded.username);
+    if (!existingJwt && existingUsers.length === 0) {
+      // First user registration, set as admin
+      sessionStorage.setItem('getIsAdmin', 'true');
+    } else {
+      decoded = jwtDecode(jwt);
+      sessionStorage.setItem('getIsAdmin', decoded.isAdmin);
     }
-    cb();
+
+    sessionStorage.setItem('jwt', jwt);
+    sessionStorage.setItem('username', decoded.username);
+  }
```

Comment:
```suggestion
        sessionStorage.setItem('jwt', jwt);

        let decoded = jwtDecode(jwt);
        sessionStorage.setItem('username', decoded.username);
        sessionStorage.setItem('isAdmin', decoded.type === 'admin'); // If the assigned type from the backend is "admin" then this will be true
  }
```